src/main/java/com/spotify/reaper/service/SegmentRunner.java

```diff
@@ -223,6 +225,7 @@ private void abort(RepairSegment segment, JmxProxy jmxConnection) {
   @Override
   public void handle(int repairNumber, ActiveRepairService.Status status, String message) {
     synchronized (condition) {
+      Thread.currentThread().setName(clusterName);
```

Comment:
I had to do it in two places because handle is called from a different thread (some JMX stuff) than the rest of the code (reaper stuff).